.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure we need this. Can you figure out what this is used for, and then evaluate whether or not we should keep/delete it?